src/app/server-cluster/ServerClusterContext.h

```diff
@@ -18,7 +18,7 @@
 
 #include <app/data-model-provider/Context.h>
 #include <app/data-model-provider/Provider.h>
-#include <lib/core/CHIPPersistentStorageDelegate.h>
+#include <app/util/persistence/AttributePersistenceProvider.h>
```

Comment:
including from `/util/` is not allowed in this - the intent is to isolate the implementation of the provider (i.e. ember or code-driven) from the server cluster implementation.